examples/buttons/bower.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "example",
+  "private": true,
+  "dependencies": {
+    "purescript-styled-components": "83d0e4f34c33375ebb78c1e0d32e34e734a7e1ff"
```

Comment:
Try to remove this and take the same approach as https://github.com/slamdata/purescript-halogen/blob/652d8de818f500062395880a320590eb71542d79/package.json#L8